### ISSUE TITLE
The service dies with "LockException".

### ISSUE THREAD

**Issue by piiskop** [traccar/traccar-web#811]
**Describe the bug**
My _nginx_ had died meanwhile. So I investigated what could cause it. I took a look into _/var/log/nginx/error.log_ and saw repeatedly errors like:
```
2021/01/25 22:41:12 [error] 223537#223537: *6071 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: traccar.tennis24.ee, request: "POST /?id=116543&timestamp=1611419171&lat=59.389104&lon=24.5806051&speed=0.0&bearing=0.0&altitude=62.53912353515625&accuracy=37.35499954223633&batt=54.0 HTTP/1.1", upstream: "http://127.0.0.1:8082/?id=116543&timestamp=1611419171&lat=59.389104&lon=24.5806051&speed=0.0&bearing=0.0&altitude=62.53912353515625&accuracy=37.35499954223633&batt=54.0", host: "traccar.tennis24.ee"
```
So I checked _traccar_'s service status and saw this:
```
kalmer@test:~$ sudo systemctl status traccar
[sudo] password for kalmer: 
● traccar.service - traccar
     Loaded: loaded (/etc/systemd/system/traccar.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Mon 2021-01-25 17:07:52 EET; 5h 34min ago
    Process: 223481 ExecStart=/opt/traccar/jre/bin/java -jar tracker-server.jar conf/traccar.xml (code=exited, status=1/FAILURE)
   Main PID: 223481 (code=exited, status=1/FAILURE)

jaan  25 17:07:52 test.tennis24.ee traccar[223481]: Caused by: liquibase.exception.LockException: Could not acquire change log lock.  Currently locked by 7595-fe17-cbff-7a2b-3480-4ec0-07d0-2001.sta.estpak.ee (2001:7d0:4ec0:3480:7a2b:cbff:fe17:7595%eth0) since 1/23/21, 7:>
jaan  25 17:07:52 test.tennis24.ee traccar[223481]:         at liquibase.lockservice.StandardLockService.waitForLock(StandardLockService.java:236)
jaan  25 17:07:52 test.tennis24.ee traccar[223481]:         at liquibase.Liquibase.clearCheckSums(Liquibase.java:1455)
jaan  25 17:07:52 test.tennis24.ee traccar[223481]:         at org.traccar.database.DataManager.initDatabaseSchema(DataManager.java:310)
jaan  25 17:07:52 test.tennis24.ee traccar[223481]:         at org.traccar.database.DataManager.<init>(DataManager.java:90)
jaan  25 17:07:52 test.tennis24.ee traccar[223481]:         at org.traccar.Context.init(Context.java:292)
jaan  25 17:07:52 test.tennis24.ee traccar[223481]:         at org.traccar.Main.run(Main.java:110)
jaan  25 17:07:52 test.tennis24.ee traccar[223481]:         ... 1 more
jaan  25 17:07:52 test.tennis24.ee systemd[1]: traccar.service: Main process exited, code=exited, status=1/FAILURE
jaan  25 17:07:52 test.tennis24.ee systemd[1]: traccar.service: Failed with result 'exit-code'.
```

So I started _traccar_'s and _nginx_'s service. Some hour later, _traccar_'s service was down again - with the same error message.

**To Reproduce**
Install _traccar_'s service and use _nginx_ as a proxy for that, wait and see!

**Expected behavior**
I expect that _traccar_ just works and doesn't die. I expect that it won't attack my web server even from the other side of the grave. I expect that _nginx_ wouldn't suffer because of actions of _traccar_.

**Comment by tananaev** [comment]
How is this related to the Traccar **web** application?

**Comment by piiskop** [comment]
The web application doesn't seem to work properly as I described.

**Comment by tananaev** [comment]
Closing as the issue clearly has nothing to do with the web app.

I would recommend asking on the forum first. I see multiple issues that you have to fix before claiming that there is some issue with the software.

**Comment by piiskop** [comment]
I don't understand your answer. _/opt/traccar/jre/bin/java -jar tracker-server.jar conf/traccar.xml_ is how you start the web app, right? That _LockException_ seems to be a class inside that JAR. If you see multiple issues that I have to fix then please tell me what I have to do to get _traccar_ work and not die every then and after. Otherwise, I see no point of having that tracker at all if I have to restart the service every now and then.

**Comment by piiskop** [comment]
I can't ask anything in the forum unless I'm signed in, right? So I went to register myself for the forum. It asks for my email address. I entered and clicked _Register_. Then, it asked me to verify: _Please verify email_, so I rewrote my email address and clicked _Register_. Once again, it asked to verify. Your forum doesn't seem to work neither.

**Comment by tananaev** [comment]
When it says "Please verify email", you should open email and click on the link to verify it.
